### DISCUSSION TITLE
How to remove the CLEAR button in notification modal

### DISCUSSION THREAD

**Discussion by bytemewebdev** [filamentphp/filament#9113] [Help] [answered]
I want to remove the CLEAR button in notification modal 

![image](https://github.com/filamentphp/filament/assets/6967014/3890d8df-6b18-4f71-a51c-e3de53be0619)

**Comment by ShaungBhone** [comment]
https://filamentphp.com/docs/3.x/notifications/customizing-notifications

**Reply by bytemewebdev** [reply]
It only covers the list of notifications but not the modal headers.

**Reply by leandrocfe** [reply, marked as answer]
create this view on your project

`resources/views/vendor/filament-notifications/components/database/modal/actions.blade.php`

add this content (is the default content without the clearNotifications action)
```php
@props(['notifications', 'unreadNotificationsCount'])

<div {{ $attributes->class('mt-2 flex gap-x-3') }}>
    @if ($unreadNotificationsCount)
        <x-filament::link color="primary" tabindex="-1" tag="button" wire:click="markAllNotificationsAsRead">
            {{ __('filament-notifications::database.modal.actions.mark_all_as_read.label') }}
        </x-filament::link>
    @endif
</div>

```

Have a look: https://filamentphp.com/community/danharrin-publishing-views-in-laravel#whats-so-dangerous

*By publishing Blade views, you are asserting that the code inside that view may never change again, throughout the life of that package's major version.*

**Reply by bytemewebdev** [reply]
seems good, so it will be publishing the view then overwriting it.

**Reply by leandrocfe** [reply]
Yes, this will work

**Reply by bytemewebdev** [reply]
It worked. Thanks.
